diff -ur libconic-0.10-o/conic/conicconnection.c libconic-0.10/conic/conicconnection.c
--- libconic-0.10-o/conic/conicconnection.c	2006-10-30 00:52:48.000000000 -0700
+++ libconic-0.10/conic/conicconnection.c	2007-06-02 08:49:12.000000000 -0600
@@ -473,7 +473,7 @@
 							  ICD_DBUS_PATH,
 							  &icd_vtable,
 							  NULL)) {
-			dbus_connection_disconnect(dbus_connection);
+			dbus_connection_close(dbus_connection);
 			dbus_connection_unref(dbus_connection);
 			dbus_connection = NULL;
 		}
diff -ur libconic-0.10-o/tests/test-automatic-events.c libconic-0.10/tests/test-automatic-events.c
--- libconic-0.10-o/tests/test-automatic-events.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-automatic-events.c	2007-06-02 08:49:30.000000000 -0600
@@ -138,7 +138,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
diff -ur libconic-0.10-o/tests/test-connect.c libconic-0.10/tests/test-connect.c
--- libconic-0.10-o/tests/test-connect.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-connect.c	2007-06-02 08:48:52.000000000 -0600
@@ -91,7 +91,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
Only in libconic-0.10/tests: test-connect.c~
diff -ur libconic-0.10-o/tests/test-disconnect.c libconic-0.10/tests/test-disconnect.c
--- libconic-0.10-o/tests/test-disconnect.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-disconnect.c	2007-06-02 08:49:30.000000000 -0600
@@ -127,7 +127,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
diff -ur libconic-0.10-o/tests/test-iap.c libconic-0.10/tests/test-iap.c
--- libconic-0.10-o/tests/test-iap.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-iap.c	2007-06-02 08:49:30.000000000 -0600
@@ -99,7 +99,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
diff -ur libconic-0.10-o/tests/test-proxies.c libconic-0.10/tests/test-proxies.c
--- libconic-0.10-o/tests/test-proxies.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-proxies.c	2007-06-02 08:49:30.000000000 -0600
@@ -159,7 +159,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
diff -ur libconic-0.10-o/tests/test-statistics.c libconic-0.10/tests/test-statistics.c
--- libconic-0.10-o/tests/test-statistics.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-statistics.c	2007-06-02 08:49:30.000000000 -0600
@@ -157,7 +157,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
diff -ur libconic-0.10-o/tests/test-two-objects.c libconic-0.10/tests/test-two-objects.c
--- libconic-0.10-o/tests/test-two-objects.c	2006-10-03 07:33:33.000000000 -0600
+++ libconic-0.10/tests/test-two-objects.c	2007-06-02 08:49:30.000000000 -0600
@@ -132,7 +132,7 @@
 
 static void finalize_dbus(void)
 {
-	dbus_connection_disconnect(dbus_connection);
+	dbus_connection_close(dbus_connection);
 	dbus_connection_unref(dbus_connection);
 }
 
