--- libconic-0.24.1/configure.ac.orig	2019-10-03 16:58:24.488734295 +0200
+++ libconic-0.24.1/configure.ac	2019-10-03 17:14:56.920024500 +0200
@@ -54,7 +54,7 @@
 AC_SUBST(OSSO_IC_CFLAGS)
 AC_SUBST(OSSO_IC_LIBS)
 
-CONCFLAGS="-Wall -Werror -Wmissing-prototypes"
+CONCFLAGS="-Wall -Wmissing-prototypes"
 AC_SUBST(CONCFLAGS)
 
 AC_CONFIG_FILES([Makefile \
--- libconic-0.24.1/tests/Makefile.am.orig	2019-10-03 16:58:24.492067610 +0200
+++ libconic-0.24.1/tests/Makefile.am	2019-10-03 17:15:33.416493448 +0200
@@ -12,8 +12,7 @@
 	@DBUS_LIBS@ \
 	$(top_builddir)/conic/libconic.la
 
-AM_LDFLAGS = -static
-AM_CFLAGS = -Wall -Werror -Wmissing-prototypes
+AM_CFLAGS = -Wall -Wmissing-prototypes
 
 bin_PROGRAMS = test-connect test-disconnect test-automatic-events \
 		test-statistics test-proxies test-iap test-two-objects \
